rollbar/__init__.py

```diff
@@ -609,6 +605,37 @@ def _report_exc_info(exc_info, request, extra_data, payload_data, level=None):
     return data['uuid']
 
 
+def _walk_trace_chain(cls, exc, trace):
+    trace_chain = [_trace_data(cls, exc, trace)]
+
+    while True:
+        exc = getattr(exc, '__cause__', None) or getattr(exc, '__context__', None)
+        if not exc:
+            break
+        trace_chain.append(_trace_data(type(exc), exc, getattr(exc, '__traceback__', None)))
+
+    return trace_chain
+
+
+def _trace_data(cls, exc, trace):
+    # exception info
+    # most recent call last
+    raw_frames = traceback.extract_tb(trace)
```

Comment:
Seemingly this would never happen if one of `__cause__` or `__context__` exist so not an issue.